src/components/Plugin/Plugin.tsx

```diff
@@ -11,15 +11,26 @@ export interface PluginProps extends ComponentPropsWithoutRef<'div'> {
   title: string;
   subtitle?: string;
   actions?: ReactNode[];
+  description?: string;
```

Comment:
I think we should have a **content** attribute only. The content can accepts **actions** type (ReactNode[]) or **description** type (string). 

Then, in function of our content we show `<PluginDescription>` or `<PluginActions>`

Reply:
i see ! clever

Reply:
![Screenshot from 2023-06-12 16-45-01](https://github.com/massalabs/ui-kit/assets/90157528/b3654202-3503-453b-8504-25a13bf96086)

So setting types created a problem that is bypassed using :any type it is not optimal but there seems to not be a fix for this situation

Reply:
I don't understand what you mean :/